WordPress/src/main/java/org/wordpress/android/ui/prefs/RelatedPostsDialog.java

```diff
@@ -92,7 +92,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
 
         toggleViews(mShowRelatedPosts.isChecked());
 
-        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity(), R.style.Calypso_AlertDialog);
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity(), R.style.Calypso_Dialog);
```

Comment:
For the ones that used to rely on `Calypso_AlertDialog` I notice that we're not using `ContextThemeWrapper` - is that on purpose?

Reply:
Nope, that was not intentional.  I must have missed those initially or during a merge conflict resolution.  Thanks for pointing it out!  I updated them in 5c1005b, ac551a0, 7ec2ab4, 770c6b4, 3a52c6c, 981c42e, 5b6dd26, ae7d2ca, and feca165.